### PR TITLE
Feat: Added Modal and Rating

### PR DESCRIPTION
**Pull request by sainath7878**
Added Modal and Rating Components

Modal:
Screenshot 2022-02-20 at 2 40 17 AM

Rating:
Screenshot 2022-02-20 at 2 41 16 AM

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: component/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.modal{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+
+    width: 60%;
+    background-color: var(--MODAL-BACKGROUND-COLOR);
+    padding: 2rem;
```
Comment (partha8) [inline component/modal/modal.css:8]: nice that you structured the code as sudesh mentioned, I have to do it as well :sweat_smile:

**Review by partha8 — APPROVED** [review]
Looks good, good work!

File: component/modal/modal.css (hunk 1 above)
Comment (sainath7878) [reply component/modal/modal.css:8]: Making a habit of following this.

**Pull request merged**